packages/build-config/package.json

```diff
@@ -37,6 +37,7 @@
     "babel-preset-env": "^1.6.0",
     "babel-preset-react": "^6.24.1",
     "caniuse-lite": "^1.0.30000751",
+    "core-js": "^2.5.1",
```

Comment:
Yes, it is a dependency of `babel-polyfill`, but since it also is a direct dependency of our project: https://github.com/xing/hops/pull/262/files#diff-94ce318d122f787a5e578e6c7c93525fR25 I decided to explicitly list it here, to not run into issues when it might be in a nested `node_modules` folder